fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -227,12 +230,14 @@ public class Loan extends AbstractAuditableWithUTCDateTimeCustom<Long> {
     private Integer termFrequency;
 
     @Setter()
+    @Enumerated(EnumType.ORDINAL)
     @Column(name = "term_period_frequency_enum", nullable = false)
-    private Integer termPeriodFrequencyType;
+    private PeriodFrequencyType termPeriodFrequencyType;
 
     @Setter(AccessLevel.PACKAGE)
+    @Enumerated(EnumType.ORDINAL)
```

Comment:
Why to use @Enumerate(ORDINAL) if converter is to be used here? Loan status enum ordinal does not follow the original behaviour...

Reply:
I can remove it form enums where I ended up using a converter, I agree that they can be confusing. The converters take precedence though, so in these cases it's still the integer values that are saved, not the ordinal values.